### ISSUE TITLE
broken forge

### ISSUE THREAD

**Issue by Nykrdaudr** [X-Niter/DTM-Integrations#30]
placing the items into a forge, no longer lets you make iron ingots. now, the only thing you can melt is the iron pipes.
![image](https://user-images.githubusercontent.com/64286618/221886120-c3a57cce-9eb0-41ac-86f2-80e06572d136.png)

**Comment by X-Niter** [comment]
You should be able to melt it into iron ingots, and you can't do this I take it?
Only the pipes can be made it seemed. 

I'll look and see why it's now allowing you to make the iron ingots. 
From looking at the screenshot, if that press is there, it should be making iron ingots.

**Comment by Nykrdaudr** [comment]
yes, it will melt all kinds of iron (scrap, blocks, ingots, radiators +++ into pipes, if you add the ingot mold.
the ss was taken the way it is, to show that the output of melting  scrap iron = iron pipe

**Comment by Nykrdaudr** [comment]
same version also have issues with all the other 7dtm ores.
none of them will even start melting...
![image](https://github.com/X-Niter/DTM-Integrations/assets/64286618/ba748785-5f25-4f9f-b4bb-e4d93c142cf7)
